cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/RootCasException.java

```diff
@@ -12,7 +12,7 @@
  * see org.jasig.cas.authentication.handler.AuthenticationException
  * @since 4.0.0
  */
-public abstract class RootCasException extends Exception {
+public abstract class RootCasException extends RuntimeException {
```

Comment:
Checked exceptions in general, I think, are a bad idea in any language. I see no benefit really. They force us to do a bunch of forced catch blocks, and potentially cause signature changes in APIs.